### PR TITLE
Clarified key concepts and purpose of sample app

### PR DESCRIPTION
**Pull request by terichadbourne**
This addresses all requests in issue 53 that apply to this particular implementation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 [![Build Status](https://travis-ci.org/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb.svg?branch=master)](https://travis-ci.org/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb)
 ![IBM Cloud Deployments](https://metrics-tracker.mybluemix.net/stats/a5cfd5b55da07a4ac1eb9590fbcc4bd3/badge.svg)
 
-# Create an Offline First Shopping List with vanilla JavaScript and PouchDB 
+# Create an Offline First Shopping List with JavaScript (aka Vanilla JS) and PouchDB 
 
-In this code pattern, we will create an Offline First shopping list. Shopping List is an Offline First demo Progressive Web App built using vanilla JavaScript and [PouchDB](https://pouchdb.com/). [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list) 
+This code pattern is a reference implementation of an Offline First shopping list app, built as a Progressive Web App using JavaScript (affectionately referred to as Vanilla JS) and [PouchDB](https://pouchdb.com/). [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list)
```
Comment (bradley-holt) [inline README.md]: perhaps:

>…using plain JavaScript…

File: README.md [hunk 2]
```diff
@@ -31,8 +31,38 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [JavaScript](https://developer.mozilla.org/bm/docs/Web/JavaScript) - this demo uses no frameworks, just plain vanilla JavaScript.
 * [Apache CouchDB](http://couchdb.apache.org/) - modern, document database hosted on your server or in the cloud.
 
+## Key concepts
+
+This shopping list app is a small single page web application consisting of an HTML file, a couple of CSS files, and a couple of JavaScript files, the PouchDB library but no other JavaScript frameworks (i.e., jQuery, Ember.js, etc.). The web page will allow multiple shopping lists to be created (e.g., Groceries, Clothes, etc.) each with a number of shopping list items associated with them (e.g., Bread, Water, etc.).
```
Comment (bradley-holt) [inline README.md]: Maybe:

>…and PouchDB. No other JavaScript framework framework (such as jQuery or Ember.js) or library other than PouchDB is used.

File: README.md [hunk 3]
```diff
@@ -31,8 +31,38 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+
+So what sets this app apart? Its Offline First architecture. The Offline First approach plans for the most constrained network environment first, enabling a great user experience even while the device is offline or has only an intermittent connection, and providing progressive enhancement as network conditions improve. This design also makes the app incredibly performant (fast!) on the best of networks. 
+
+PouchDB, CouchDB, and Service Worker are the primary tools that turn our simple shopping list app into a high performance, offline-capable Progressive Web App.
+
+**Data stays safe on your device, even while it's offline.**  <br>Persistance of shopping lists and item data entered by the user is achieved using the in-browser database PouchDB. This will allow your data to survive between sessions and when disconnected from the network. (Whether you remember that you need juice while you're on your trusty home WiFi or in the middle of the wilderness, you can still add it your list.)
```
Comment (bradley-holt) [inline README.md]: Markdown formatting note: You can do a newline by entering two spaces at the end of the previous line and hitting enter.
https://daringfireball.net/projects/markdown/syntax#p

Comment (bradley-holt) [inline README.md]: "Wi-Fi"

File: README.md [hunk 4]
```diff
@@ -31,8 +31,38 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+
+# Live demo 
+
+To see this app in action without installing anything, simply visit https://ibm-watson-data-lab.github.io/shopping-list-vanillajs-pouchdb/ in a web browser or on your mobile device.
+
 # Steps
 
+Want to check out the end product on your own machine? Follow these steps to deploy your own instance of the shopping list app.
+
+**THIS PARAGRAPH NEEDS HELP FROM SOMEONE WHO UNDERSTANDS THESE OPTIONS.** <BR>If X, you may want to deploy the app to the IBM Cloud. Alternatively, you may run it locally using your own CouchDB instance or the hosted Cloudant service.
```
Comment (bradley-holt) [inline README.md]: Maybe:

>This app can be deployed to IBM Cloud. You can also run this app on your local machine for development purposes using either a local Apache CouchDB instance or an IBM Cloudant service instance from the IBM Cloud Catalog.

Comment (terichadbourne) [reply README.md]: This still doesn't help me understand what the options mean. 

What's the difference between the first sentence and the second half of the last sentence? In the first your HTML file lives on Bluemix and you use Cloudant, and in the latter your HTML file is on your laptop and you use Cloudant?

**Comment by terichadbourne** [comment]
My latest commit addresses (I think) all of @bradley-holt's edits.

**Pull request merged**